src/main/java/tests/testCase.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+import loginPage.Homepage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class testCase {
+    WebDriver driver = new FirefoxDriver();
+    Homepage page;
+    @BeforeMethod
+    public void browserSetup() {
+        page = new Homepage(driver);
```

Comment:
If you had a login page or a my account page, what would you name it? This should be instead `home = new HomePage(driver);`